temporian/core/operators/assign.py

```diff
@@ -22,34 +24,50 @@
 
 
 class AssignOperator(Operator):
-    """Simple moving average operator."""
+    """Assign operator."""
 
     def __init__(
         self,
-        left_event: Event,
-        right_event: Event,
+        event_1: Event,
+        event_2: Event,
+        event_3: Optional[Event] = None,
+        event_4: Optional[Event] = None,
```

Comment:
Why do we have 2 optional events to assign and not a list of optional events?

Reply:
Agree! using `*args` here to accept an arbitrary number of events would both improve the API and simplify the code below

Reply:
Good point.
*Done